core/src/main/java/org/jdbi/v3/core/extension/ExtensionHandler.java

```diff
@@ -52,25 +52,26 @@ public interface ExtensionHandler {
      * @param args Optional arguments for the handler
      * @return The return value for the method that was bound to the extension handler. Can be null
      * @throws Exception Any exception from the underlying code
+     * @deprecated call {@link #createInvoker(Object)} instead
      */
-    Object invoke(HandleSupplier handleSupplier, Object target, Object... args) throws Exception;
+    @Deprecated(forRemoval = true)
```

Comment:
I left this only to try to satisfy `japicmp` but it still seems to be unhappy and I do not understand why.

I am tempted to delete this entirely and just ignore the japicmp failure since this is `@Alpha`.

Reply:
https://github.com/siom79/japicmp/issues/421

Reply:
You can remove this and add the exclude statement to the pom. This is temporarily and can go away once we have done another release.